src/nile/signer.py

```diff
@@ -0,0 +1,48 @@
+"""Utility for sending signed transactions to an Account on Starknet."""
+import subprocess
+
+from starkware.cairo.common.hash_state import compute_hash_on_elements
```

Comment:
This should be be wrapped in a try catch to avoid throwing when users don't have cairo installed yet (and want to use nile to install it).